index.html

```diff
@@ -2,306 +2,293 @@
+                            <li>Html</li>
+                            <li>Css</li>
+                        </ul>
+                    </div>
+                    <div class="skill-content">
+                        <div class="title">
+                            <img src="./assets/images/sqaure.png" alt="img" />
+                            <h1>Frameworks</h1>
+                        </div>
+                        <ul class="frame1">
+                            <li>Bootstrap</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSPec</li>
+                            <li>CapyBara</li>
+                            <li>Selenium</li>
+                        </ul>
+                    </div>
 
+                    <div class="skill-content">
+                        <div class="title">
+                            <img id="skill-img" src="./assets/images/circle.png" alt="img" />
+                            <h1 id="skill-text">Skills</h1>
+                        </div>
+                        <ul class="skills1">
+                            <li>Codekit</li>
+                            <li>Github</li>
+                            <li>Codepen</li>
+                            <li>Gitlab</li>
+                            <li>Terminal</li>
+                        </ul>
+                    </div>
+                </div>
 
-      </div>
-    </div>
-  </section>
-  <section class="contact-section">
-    <div class="contact-me">
-      <p>
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </p>
-      <form class="form" action="https://formspree.io/f/xrgjdpow" method="POST">
-        <input type="text" name="name" class="text" placeholder="Full name" maxlength="30" required />
-        <input type="email" class="text" placeholder="Email address" required />
-        <textarea name="comment" class="input-text" placeholder="Enter text here" maxlength="500"></textarea>
-        <button type="submit" class="button">Get in touch</button>
 
-      </form>
-    </div>
-  </section>
+            </div>
+        </div>
+    </section>
+    <section id="contact_section" class="contact-section">
+        <div class="contact-me">
+            <p>
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form class="form" action="https://formspree.io/f/xrgjdpow" method="POST">
+                <input type="text" name="name" class="text" placeholder="Full name" maxlength="30" required />
+                <input type="email" class="text" placeholder="Email address" pattern="(?=.*[a-z])" title="Must contain lowercase letters!" required />
+                <textarea name="comment " class="input-text " placeholder="Enter text here " maxlength="500"></textarea>
+                <button type="submit" class="button">Get in touch</button>
 
+            </form>
```

Comment:
- [ ] Please fix that as even when the user enters an email in a small case the form doesn't submit.
- [ ] Kindly be noticed that the error message should show up close to the "submit" button.